src/main/java/uk/gov/hmcts/darts/hearings/service/impl/HearingsServiceImpl.java

```diff
@@ -69,4 +80,17 @@ private List<TranscriptionEntity> findNonAutomaticTranscripts(List<Transcription
             .toList();
     }
 
+    @Override
+    public List<Annotation> getAnnotationsByHearingId(Integer hearingId) {
+        List<AnnotationEntity> annotations;
+        if (authorisationApi.userHasOneOfRoles(ADMIN_ROLE)) {
+            //admin will see all annotations
+            annotations = annotationRepository.findByHearingId(hearingId);
+        } else {
+            //judge will only see their own annotations
```

Comment:
Consider rewording this comment to "others will only see their own annotations", as chances are this comment will never be updated if the allowable roles on the controller are ever broadened.

Reply:
done